globals/utility_globals.go

```diff
@@ -0,0 +1,32 @@
+package common_globals
+
+import (
+	"database/sql"
+
+	"github.com/PretendoNetwork/nex-go/v2"
+	"github.com/PretendoNetwork/nex-go/v2/types"
+)
+
+// UtilityManager manages NEX unique ids
```

Comment:
```suggestion
// UtilityManager manages NEX unique IDs
```

Please make *all* occurences of "id" uppercase

Reply:
Fixed in latest commit (at least, I hope it is 😭)

Reply:
You didn't change this one, so definitely not all occurrences have been fixed 😅